README.md

```diff
@@ -151,12 +151,10 @@ Wrangler can be installed both through [npm](https://www.npmjs.com/get-npm) and
 
 ### Using `npm`:
 
-1. If you don't already have npm on your machine, install it. On OSX with [Homebrew](https://brew.sh/) that looks like:
+1. If you don't already have npm on your machine, install it using [npm's recommended method](https://www.npmjs.com/get-npm).
```

Comment:
we should specify that this is using a node version manager and list/link some here. i think linking out to npm is good but i don't want to force people to click that link to get to where they need to go.

Reply:
i think it's important to explain that an OS package manager will install node and it's global node_modules dir in a way that is only accessible to root. using the version manager, it will scope the node installation to the user and as a result will eliminate the need for root permissions

Reply:
I had this before: 

```
If you don't already have npm on your machine, install it using npm's recommended method: [a version manager](https://www.npmjs.com/get-npm).
```

And reverted it at @ashleymichal suggestion as I felt it was clearer without it, and the npm docs are fairly comprehensive wrt installing node.

Reply:
the first thing you see when you scroll down the get-npm docs (which fwiw i wrote, many years ago) is this:

<img width="703" alt="Screen Shot 2019-06-28 at 3 35 25 PM" src="https://user-images.githubusercontent.com/1163554/60366732-68a32580-99ba-11e9-8183-6292f5a310f1.png">

we don't want folks to do that. we need to make it clear that the version manager strategy is the one to pick.

Reply:
ahhh I see. Will fix this